components/scripts/lib/gradle.sh

```diff
@@ -26,8 +26,13 @@ invoke_gradle() {
     args+=("-Pcom.gradle.enterprise.build_validation.server=${ge_server}")
   fi
 
+  if [ "$build_scan_publishing_mode" == "off" ]; then
```

Comment:
I would invert this if-statement and handle `on` case first. Just like you do further up in `print_gradle_command()`.

Reply:
Changed in 17fe64ac094a951f47340572dc60051a9419a40d